tuf-spec.md

```diff
@@ -1470,18 +1486,53 @@ it in the next step.
   2. Otherwise, recursively search the list of delegations in
      order of appearance.
 
-    1. If the current delegation is a multi-role delegation,
+    1. Let DELEGATEE denote the current target role TARGETS is
```

Comment:
Could we please say DELEGATOR instead of TARGETS? Can be conflated at first glance with the top-level targets role.

Reply:
This is using the same `TARGETS` defined at the beginning of step 7 (line 1475 in this version). I'll see if I can clarify this.

Reply:
Sorry, just to be sparkling clear, I think we should start off by setting the DELEGATOR role to the top-level targets role, and go from there. Otherwise, it could confuse a few readers.

Reply:
better?

Reply:
I think it could be clearer. Please see my latest suggestions?

Reply:
That makes sense. I think yours maps more closely to what the code would look like.